### DISCUSSION TITLE
Inter-Panel communication

### DISCUSSION THREAD

**Discussion by samant-rags** [gapitio/gapit-htmlgraphics-panel#57] [Q&A]
Suppose if i have 2 Grafana panels. Eg. Search Panel and Results Panel. Can i pass data around between the 2 panels using this plugin ?

**Comment by ZuperZee** [comment]
I made an [example](https://gapit-htmlgraphics-panel.gapit.io/docs/examples/communicate-between-panels/) on the website on how it can be implemented. There is currently a bug with onpanelwillunmount #58, but it shouldn't affect the code for the users. _Might be a bit annoying when editing a lot tho._
